crates/rpc/fixtures/0.6.0/transactions/status_l2_accepted.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
These statuses didn't change with protocol version, right? IMHO the test would be more readable / informative if it used the same fixture for all versions...

Reply:
The way the macro is done (using `include_str!`) requires us to have a file for every version on these multi-version tests. It's a bit messy otherwise, I just did it once in one of the `simulate` methods which is just for v08.

Reply:
Obviously you don't have to use `assert_json_matches_fixture` when the fixture doesn't change - in fact, I'm arguing that you shouldn't...

Reply:
Then you're missing the whole point of the PR

Reply:
The point of the PR is not to have the maximum number of fixtures, but to test JSON-RPC methods for all supported JSON-RPC versions. Obviously one can do that without `assert_json_matches_fixture`.